docker/1.6.0/py2/Dockerfile.cpu

```diff
@@ -56,8 +86,17 @@ RUN pip install --no-cache --upgrade \
     scipy==1.2.2 \
     urllib3==1.24 \
     python-dateutil==2.8.0 \
+    mpi4py==3.0.2 \
+    gluoncv==0.6.0 \
     ${MX_URL} \
-    sagemaker-mxnet-training==3.1.6
+    sagemaker-mxnet-training==3.1.6 \
+ && pip install --no-cache-dir -U \
+    horovod==0.19.0 
```

Comment:
does this need to be done in it's own pip install statement?

Reply:
I have followed the TF dockerfile to make it as uniform as possible. Anyway is fine.

Reply:
Adding a Horovod installation on a separate line makes more sense. Done. [540472](https://github.com/aws/sagemaker-mxnet-container/pull/133/commits/5404725498a439c43d4d8111866a97d7083153c9)